kdbg/.gitignore

```diff
@@ -6,3 +6,7 @@ moc_*.cpp
 ui_*.h
 version.h
 pics/sc-*.svgz
+
+#KDevelop files
```

Comment:
Please put files that your personal development environment produces into `.git/info/excludes`. They do not go into `.gitignore`.